### PR TITLE
Add: SparkLabs.openvpn-configuration-generator version 1.0.8.1008

### PR DESCRIPTION
**Pull request by quhxl**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20343)

### CODE REVIEW

**Comment by quhxl** [comment]
Screenshot_205

File: manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: SparkLabs.openvpn-configuration-generator
+PackageVersion: 1.0.8.1008
+PackageLocale: en-US
+Publisher: SparkLabs Pty Ltd
+PublisherUrl: https://www.sparklabs.com
+PublisherSupportUrl: https://www.sparklabs.com/support
+PrivacyUrl: https://www.sparklabs.com/company/legal/privacy
```
Comment (OfficialEsco) [inline manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml:10]:
```suggestion
PrivacyUrl: https://www.sparklabs.com/company/legal/#privacy
```
This is 100% my fault, my YamlCreate script removes all `#` 🤦‍♂️ and i don't really know how to work around it

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
URI: https://www.sparklabs.com, Validation result: Passed
URI: https://www.sparklabs.com/support, Validation result: Passed
URI: https://www.sparklabs.com/company/legal/privacy, Validation result: Failed, Http status code: NotFound 
URI: https://github.com/thesparklabs/openvpn-configuration-generator, Validation result: Passed
URI: https://github.com/thesparklabs/openvpn-configuration-generator/blob/master/LICENSE, Validation result: Passed
URI: https://github.com/thesparklabs/openvpn-configuration-generator/releases/download/1.0.8/OpenVPNConfigurationGeneratorx64_Win10_1.0.8.msi, Validation result: Passed
URI: https://github.com/thesparklabs/openvpn-configuration-generator/releases/download/1.0.8/OpenVPNConfigurationGeneratorx86_1.0.8.msi, Validation result: Passed
```

File: manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml (hunk 1 above)
Comment (OfficialEsco) [reply manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml:10]: Oke.. Solved it🤦‍♂️
`$Variable = $Line.Replace("#","").Split(":").Trim()` -> `$Variable = $Line.TrimStart("#").Split(":").Trim()`
imma try to get some more changes in

Comment (quhxl) [reply manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml]: That's nice!

Comment (OfficialEsco) [reply manifests/s/SparkLabs/openvpn-configuration-generator/1.0.8.1008/SparkLabs.openvpn-configuration-generator.locale.en-US.yaml]: #20359 actually fixes more issues than expected 🎉

**Comment by palenshus** [comment]
The issue here is that the tool ends with exit code: 9009, which the validator takes as a failure. Ideally the exe would return 0 when run with no arguments, otherwise we require a validation override.

**Review by OfficialEsco — APPROVED** [review]
@palenshus you need to add `Azure-Pipeline-Passed` and `Moderator-Approved`

**Comment by denelon** [comment]
@OfficialEsco we're getting ready to add another feature for validation. We're going to add a .validation file that contains exceptions so we can have the "future" PRs for a package pass if they fail for the same reason as a previous manifest. The hope is that when we see this exact same error again, we won't have to manually intervene. Of course in some cases we may not automatically pass if other things change and cause a fail, but it should help in this case. We're testing the feature now, so hopefully we will be able to release it in the next couple of weeks.

**Review by denelon — APPROVED** [review]

**Pull request merged**